prom_sky_con/README.md

```diff
@@ -0,0 +1,97 @@
+# Overview
```

Comment:
just provide an overview and unique parameters, no need to repeat info that is generic and applies to all connectors

Reply:
It is likely that this prometheus-skydive-connector will be used stand-alone, where the user has no idea about how to run skydive and the skydive-flow-exporter. It is therefore important for self-contained instructions to be available, and the reviewers of the project for which this was developed will look for such instructions to accompany the code.

Reply:
no happy the duplication, but leaving to your discretion